CONTRIBUTING.md

```diff
@@ -0,0 +1,158 @@
+# Contribute to WarpX
+
+We welcome new contributors! Here is how to participate to the WarpX 
+development.
+
+## Git workflow
+
+The WarpX project uses [git](https://git-scm.com) for version control. If you 
+are new to git, you can follow one of these tutorials:
+- [Learn git with bitbucket](https://www.atlassian.com/git/tutorials/learn-git-with-bitbucket-cloud)
+- [git - the simple guide](http://rogerdudler.github.io/git-guide/)
+
+### Create your branch from the `dev` branch
+
+Follow instructions from the 
+[WarpX documentation](https://ecp-warpx.github.io/doc_versions/dev/building/building.html) 
+to install the code. Make sure you are on WarpX `dev` branch with
+```
+git checkout dev
+```
+in the WarpX directory.
+
+Create a branch `<branch_name>` (the branch name should reflect the piece 
+of code you want to add, like `fix_spectral_solver`) with
+```
+git checkout -b <branch_name>
+```
+and do the coding you want. It is probably a good time to look at the 
+[AMReX documentation](https://amrex-codes.github.io/amrex/docs_html/). 
+Add the files you work on to the git staging area with 
+```
+git add file_I_created and_file_I_modified
+```
+### Commit & push your changes
+
+Periodically commit your changes with
+```
+git commit -m "This is a 50-char description to explain my work"
+```
+
+The commit message (between quotation marks) is super important to follow the 
+developments and identify bugs.
+
+For the moment, commits are on your local repo only. You can push them to 
+the WarpX Github repo with
+```
+git push -u origin <branch_name>
+```
```

Comment:
Well, this requires that the person doing `git push` has write permissions to the WarpX repo. (which, for the moment, is restricted to ~8 WarpX developers). 

I would suggest to explain how to create a Fork, as this does not require Write permissions. It seems that this is also what [amrex suggests](https://github.com/AMReX-Codes/amrex/blob/master/README.md#development-model) for their development model.

I know this complicates a bit the workflow ... We could of course add a remark, saying that someone who is part of the WarpX development team (i.e. has write access to the repo) does not need to create a Fork.

Reply:
I agree, I'll update the section